app/src/components/HelpView.vue

```diff
@@ -0,0 +1,127 @@
+<style lang="scss">
+  .c-help {
+    position : relative;
+
+    padding : 1em;
+
+    text-align : center;
+
+    &--definitionList {
+      font-size : .9em; 
+
+      dt {
+        clear   : both;
+        width   : 20%;
+        padding : .5em 0;
+      }
+
+      dd {
+        width      : 80%;
+        text-align : left;
+
+        padding : .5em 0 .5em .5em;
+      }
+
+      dt, dd {
+        float   : left;
+        margin  : 0;
+      }
+    }
+
+    &--key {
+      display :  inline-block;
+
+      background-color : var(--header-background);
+      border-radius    : 2px;
+
+      color : var( --code-color );
+
+      padding : .25em .5em;
+
+      text-align : center;
+
+      min-width : 2em;
+      height    : 2em;
+    }
+  }
+</style>
+
+<template>
+  <div class="c-help">
+    <div class="c-dragHandle"></div>
+
+    <div class="c-logo" transition="project">
+      <svg width="330" height="330" viewBox="0 0 330 330" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+        <title>Forrest logo</title>
+        <defs>
+          <linearGradient x1="50%" y1="0%" x2="50%" y2="100%" id="c">
+            <stop stop-color="#EEEBEB" offset="0%" />
+            <stop stop-color="#FCFBFB" offset="18.142%" />
+            <stop stop-color="#CFCFCF" offset="100%" />
+          </linearGradient>
+          <ellipse id="b" cx="161.175" cy="161.175" rx="161.175" ry="161.175" />
+          <filter x="-50%" y="-50%" width="200%" height="200%" filterUnits="objectBoundingBox" id="a">
+            <feOffset dy="1" in="SourceAlpha" result="shadowOffsetOuter1" />
+            <feGaussianBlur stdDeviation="1" in="shadowOffsetOuter1" result="shadowBlurOuter1" />
+            <feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.5 0" in="shadowBlurOuter1" />
+          </filter>
+          <linearGradient x1="50%" y1="0%" x2="50%" y2="100%" id="d">
+            <stop stop-color="#344352" offset="0%" />
+            <stop stop-color="#2A333C" offset="100%" />
+          </linearGradient>
+          <ellipse id="e" cx="161.175" cy="161.175" rx="150.059" ry="150.059" />
+          <filter x="-50%" y="-50%" width="200%" height="200%" filterUnits="objectBoundingBox" id="f">
+            <feGaussianBlur stdDeviation="1.5" in="SourceAlpha" result="shadowBlurInner1" />
+            <feOffset dy="1" in="shadowBlurInner1" result="shadowOffsetInner1" />
+            <feComposite in="shadowOffsetInner1" in2="SourceAlpha" operator="arithmetic" k2="-1" k3="1" result="shadowInnerInner1"
+            />
+            <feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.5 0" in="shadowInnerInner1" />
+          </filter>
+        </defs>
+        <g fill="none" fill-rule="evenodd">
+          <g transform="translate(4 3)">
+            <use fill="#000" filter="url(#a)" xlink:href="#b" />
```

Comment:
I see you put a lot of effort in the svg things. :)

These are the first things that use gradients and shadows... For me they're no matching 100% the interface ( 90% ) and they add more complexity to this view...

Maybe we can live without them?